src/random.rs

```diff
@@ -12,10 +12,12 @@ extern "C" {
 #[cfg(not(target_arch = "wasm32"))]
 use rand::prelude::*;
 
+/*
```

Comment:
Why is this being commented?

Reply:
I was having the following problem:

```
error[E0428]: the name `random` is defined multiple times
  --> src/random.rs:25:1
   |
16 | pub fn random() -> f64 {
   | ---------------------- previous definition of the value `random` here
...
25 | pub fn random() -> f64 {
   | ^^^^^^^^^^^^^^^^^^^^^^ `random` redefined here
   |
   = note: `random` must be defined only once in the value namespace of this module
```

commenting this out solved it but it would be better to remove it if it's not useful. Do you have a better suggestion?